FAQ.md

```diff
@@ -207,3 +208,17 @@ become visible if you hover over the button with your cursor.
 You can also use the refresh icon button on the top right of the screen to achieve
 the same result:
 ![pull-request-ci-workflow-details-page-2.png](./docs/assets/ci-single-job-re-run-tutorial/pull-request-ci-workflow-details-page-2.png)
+
+
+## 14. How to fix the script configure, stuck at resolving npm registry (ubuntu)?
+
```

Comment:
@jagpreetsinghsasan Here I highly recommend including about 10 lines worth of the logs that appear on the terminal when the problem happens. That way people can recognize and quickly decide if the FAQ answer is the solution to the same problem they are having or not.